src/components/CustomPageTitle.astro

```diff
@@ -0,0 +1,8 @@
+---
+const { title, author, draft } = Astro.props.entry.data;
+---
+
+<header>
+  <h1>{title}</h1>
+  <em>Author: {author}</em>
+</header>
```

Comment:
```suggestion
---
const { title, author } = Astro.props.entry.data;
---

<h1 id="_top">{title}</h1>
{author && (
  <em>Author: {author}</em>
)}

<style>
  h1 {
    margin-top: 1rem;
    font-size: var(--sl-text-h1);
    line-height: var(--sl-line-height-headings);
    font-weight: 600;
    color: var(--sl-color-white);
  }
</style>
```

Let's conditionally render the author. This also restores default styling to the h1, both by applying manually and removing the wrapping <header>